var/spack/repos/builtin/packages/bzip2/package.py

```diff
@@ -160,3 +160,21 @@ def install(self, spec, prefix):
                 force_remove("bunzip2", "bzcat")
                 symlink("bzip2", "bunzip2")
                 symlink("bzip2", "bzcat")
+
+    @run_after("install")
+    def install_pkgconfig(self):
+        pkg_path = join_path(self.prefix.lib, "pkgconfig")
+        mkdirp(pkg_path)
+
+        with open(join_path(pkg_path, "zlib.pc"), "w") as f:
+            f.write("prefix={0}\n".format(self.prefix))
+            f.write("exec_prefix=${prefix}/bin\n")
+            f.write("libdir=${prefix}/lib\n")
+            f.write("includedir=${prefix}/include\n")
```

Comment:
This should at least use something like `self.spec["bzip2"].libs.directories[0]` for `libdir`.